README_traceability.md

```diff
@@ -16,21 +16,21 @@ docker run --env-file env_vars -v `pwd`/keys:/keys ghcr.io/axway/mulesoft_tracea
+| MULESOFT_PROXYURL               | mulesoft.proxyUrl               | The url for the proxy for API Manager (e.g. <http://username:password@hostname:port>). If empty, no proxy is defined.                                                                                                                                                                        | Internally, this value defaults to empty                                                                                                                                          |
+| MULESOFT_SSL_INSECURESKIPVERIFY | mulesoft.ssl.insecureSkipVerify | InsecureSkipVerify controls whether a client verifies the server's certificate chain and host name. If InsecureSkipVerify is true, TLS accepts any certificate presented by the server and any host name in that certificate. In this mode, TLS is susceptible to man-in-the-middle attacks. | Internally defaulted to false                                                                                                                                                     |
+| MULESOFT_SSL_CIPHERSUITES       | mulesoft.ssl.cipherSuites       | An array of strings. It is a list of supported cipher suites for TLS versions up to TLS 1.2. If CipherSuites is nil, a default list of secure cipher suites is used, with a preference order based on hardware performance.                                                                  | [See](https://docs.axway.com/bundle/amplify-central/page/docs/connect_manage_environ/connected_agent_common_reference/agent_security/index.html) for default cipher suite setting |
+| MULESOFT_SSL_MAXVERSION         | mulesoft.ssl.maxVersion         | String value for the maximum SSL/TLS version that is acceptable. If empty, then the maximum version supported by this package is used, which is currently TLS 1.3. Allowed values are: TLS1.0, TLS1.1, TLS1.2, TLS1.3                                                                        | Internally, this value defaults to empty                                                                                                                                          |
+| MULESOFT_SSL_MINVERSION         | mulesoft.ssl.minVersion         | String value for the minimum SSL/TLS version that is acceptable. If zero, empty TLS 1.0 is taken as the minimum. Allowed values are: TLS1.0, TLS1.1, TLS1.2, TLS1.3                                                                                                                          | Internally, the value defaults toTLS1.2                                                                                                                                           |
```

Comment:
unless its not rendering right for me in this view, there needs to be a space between to and TLS1.2 (toTLS1.2)